Content.Shared/Storage/Components/SecretStashComponent.cs

```diff
@@ -7,54 +7,89 @@
 using Robust.Shared.GameStates;
 using Content.Shared.DoAfter;
 using Robust.Shared.Serialization;
+using Robust.Shared.Audio;
 
 namespace Content.Shared.Storage.Components
 {
     /// <summary>
     ///     Logic for a secret slot stash, like plant pot or toilet cistern.
-    ///     Unlike <see cref="ItemSlotsComponent"/> it doesn't have interaction logic or verbs.
-    ///     Other classes like <see cref="ToiletComponent"/> should implement it.
+    ///     Unlike <see cref="ItemSlotsComponent"/> it has logic for opening and closing
+    ///     the stash.
     /// </summary>
     [RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
     [Access(typeof(SecretStashSystem))]
     public sealed partial class SecretStashComponent : Component
     {
         /// <summary>
-        ///     Max item size that can be fitted into secret stash.
+        ///     Max item size that can be inserted into secret stash.
         /// </summary>
         [DataField("maxItemSize")]
         public ProtoId<ItemSizePrototype> MaxItemSize = "Small";
 
         /// <summary>
-        /// If stash has way to open then this will switch between open and closed.
+        ///     If stash has way to open then this will switch between open and closed.
         /// </summary>
         [DataField, AutoNetworkedField]
-        public bool ToggleOpen;
+        public bool IsStashOpen = false;
```

Comment:
use OpenableComponent instead

Reply:
Sorry delta I am confused on this one 😆!
I looked at the component but it appears to be for like drinks and stuff? I don't see how I could support having the verb system I put in place (E.g only appearing if your holding the correct tool). Mind explaining a little more?

Reply:
im 99% sure theres bools for closeable and openable to disable the verb for opening it, then have ToolOpenableSystem do the actual opening